playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -90,3 +90,34 @@ test('signup and highlight', async ({ page, isMobile }) => {
   highlightcount = await BookPage.highlightCount()
   expect(highlightcount).toBe(0)
 })
+
+test('multiple highlight', async ({ page, isMobile }) => {
+  test.skip(isMobile as boolean, 'test only desktop resolution')
+
+  // GIVEN: Open Rex page
+  const BookPage = new ContentPage(page)
+  const path = '/books/introduction-anthropology/pages/7-introduction'
+  await BookPage.open(path)
+
+  // AND: Signup as a new user
+  await rexUserSignup(page)
+  await expect(page).toHaveURL('/books/introduction-anthropology/pages/7-introduction')
+
+  // WHEN: Highlight any random paragraph
+  const paracount = BookPage.paracount()
+  const randomparanumber = randomNum(await paracount)
+  await BookPage.highlightText('green', randomparanumber)
+
+  // THEN: Text is highlighted
+  let highlightcount = await BookPage.highlightCount()
+  expect(highlightcount).toBe(1)
+
+  // AND: Highlight another random paragraph
+  await BookPage.scrolltotop()
+  const randomparanumber2 = randomNum(await paracount, randomparanumber)
+  await BookPage.highlightText('yellow', randomparanumber2)
+
+  // THEN: Text is highlighted
+  highlightcount = await BookPage.highlightCount()
```

Comment:
`highlightcount` works only if we create an instance again (first one created in line 112). Is that how it works?

Reply:
Yes, that would be expected.